src/test/java/Homework20.java

```diff
@@ -0,0 +1,102 @@
+        playButton.click();
+    }
+    public void soundBars() {
+        WebElement usersAvatar = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("img[alt='Sound bars']")));
+        Assert.assertTrue(usersAvatar.isDisplayed());
+    }
+
+    @Test
+    public void deletePlaylist() throws InterruptedException{
+        login("jenyag5@gmail.com", "te$t$tudent");
+        playlist();
+        verifyPlaylistDeleted();
+    }
+    public void playlist() throws InterruptedException {
+        WebElement songs = driver.findElement(By.cssSelector("#playlists > ul > li:nth-child(5) > a"));
+        songs.click();
+        WebElement delete = wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//*[@id=\"playlistWrapper\"]/header/div[3]/span/button")));
+        delete.click();
+    }
+    public void verifyPlaylistDeleted() {
+        WebElement imageDisplayed = driver.findElement(By.cssSelector("div[class='success show']"));
+        Assert.assertTrue(imageDisplayed.isDisplayed());
+    }
+
+    @Test
+    public void addSongtoPlaylist () throws InterruptedException {
+        openloginUrl();
+        enterEmail("jenyag5@gmail.com");
+        enterPassword("te$t$tudent");
+        clickSubmit();
+        allSongs();
+        songName();
+        addTo();
+        playlistAuto();
+        Assert.assertTrue(getSuccessPopUp().isDisplayed());
+    }
+    public void openloginUrl() {
+        driver.get("http://bbb.testpro.io/");
+    }
+    public void enterEmail(String email) {
+        WebElement emailField = driver.findElement(By.cssSelector("[type='email']"));
+        emailField.click();
+        emailField.sendKeys(email);
+    }
+    public void enterPassword(String password) {
+        WebElement passwordField = driver.findElement(By.cssSelector("[type='password']"));
+        passwordField.click();
+        passwordField.sendKeys(password);
+    }
+    public void allSongs() throws InterruptedException {
+        WebElement allSongs = driver.findElement(By.cssSelector("#sidebar > section.music > ul > li:nth-child(3) > a"));
+        allSongs.click();
+    }
+    public void songName() throws InterruptedException {
+        WebElement songName = wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//*[@id=\"songsWrapper\"]/div/div/div[1]/table/tr[1]/td[2]")));
+        songName.click();
+    }
+    public void addTo() throws InterruptedException {
+        WebElement addTo = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("button[class='btn-add-to']")));
+        addTo.click();
+    }
+    public void playlistAuto() throws InterruptedException {
+        WebElement playlistAuto = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("#songsWrapper > header > div.song-list-controls > div > section.existing-playlists > ul > li:nth-child(5)")));
+        playlistAuto.click();
```

Comment:
Please do not use plugins or copy xpath from page code using browser or web developer tools.